mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -2606,6 +2656,11 @@ func resourceContainerClusterRead(d *schema.ResourceData, meta interface{}) erro
             }
         }
 	}
+	if aprc := flattenAdditionalPodRangesConfig(cluster.IpAllocationPolicy); aprc != nil {
```

Comment:
It can be moved inside [flattenIPAllocationPolicy](https://github.com/GoogleCloudPlatform/magic-modules/blob/0102b22bbd392fb6c19e7908f6c08ed7be9c9904/mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb#L5643).